### DISCUSSION TITLE
Patching numbered lists

### DISCUSSION THREAD

**Discussion by lm-hmwill** [dolanmiu/docx#2732] [General]
I have been looking into inserting bulleted lists as part of patches. The underlying use case is that certain text blocks that are being patched into a document are coming in the form of markdown, which does allow for certain formatting.

For numbered lists, though, I am not sure if it's possible to create those as part of patching or not. The documentation for numbering in general (outside of patching) seems to indicate, that it is necessary to provide [numbering settings as part of Document options](https://docx.js.org/#/usage/numbering?id=configuration), which can then be referred to in the individual list items added.

However, I am not seeing a way how to add such numbering configs to the Patcher. Or is there? Alternatively, is it possible to create a document template with appropriately configured numbering configs for different levels, that one could refer to from list items contained in patches?

**Comment by douglassllc** [comment]
@lm-hmwill did you ever get a response?  Or did you figure out away of getting number list in a patch document?  I have attempted several suggestions I have found related to adding number list, style, paragraphs to my template, but none has worked.

**Reply by lm-hmwill** [reply]
No, unfortunately I haven't gotten a response and have not been able to make further progress myself.

**Reply by douglassllc** [reply]
Thank you for the reply.  I am going to keep working as time permits.  If I find a solution I will post it here on this thread.
